R/values.R

```diff
@@ -0,0 +1,52 @@
+#' Create a values object.
+#'
+#' Create a values object for use in other functions
+#'
+#' @param ... value objects. Created by measure.value() or attribute value().
+#' @examples
+#' data(example.data)
+#' config <- create.config(example.data, title = "sweden_at_eurovision_no_missing")
+#' config <- add.measure.column(config, column.name = "Measure",
+#'    values = values(measure.value(label = "Final Rank"),
+#'    measure.value(label = "Final Points"),
+#'    measure.value(label = "People on Stage")))
+#' @export
+values <- function(...) {
+  values <- list(...)
+  l <- list()
+  for (value in values) {
+    if (!inherits(value,"value")) {
```

Comment:
Again, I think we want to support users putting string values in here and then have them automatically converted into the appropriate measure/attribute value with the string as the label.